chromium_src/content/browser/browser_context.cc

```diff
@@ -39,3 +39,9 @@ std::string GetSessionStorageNamespaceId(WebContents* web_contents) {
 }  // namespace content
 
 #include "../../../../content/browser/browser_context.cc"
+
+namespace content {
+bool BrowserContext::IsTor() const {
+  return false;
+}
+}  // namespace content
```

Comment:
This is awesome 👍

Reply:
Out of curiosity, would this ever get called? BrowserContext::IsOffTheRecord is declared pure virtual and Profile's override gets called. Wouldn't IsTor be similar?

Reply:
This is intentional. If we do it pure virtual and we need to implement it for every subclasses. And that might introduces extra patches/define, so it is cleaner doing it this way.

Reply:
And yes, it would get called for those subclasses don't override it